source/includes/api/_areas_v2.md

```diff
@@ -326,46 +393,25 @@ curl -X PATCH https://api.resourcewatch.org/v2/area/:id
-subscriptionId       | The ID of the subscription associated with this area. Read-only attribute.                         | String  | 5e4d7c47dd8fa31290d548ae |
+Use this endpoint to update an existing area. For a detailed description of each field that can be provided in the body of the request, check out the [Area reference](#area-reference) section.
 
-### Email notification
+Keep in mind that you should provide one of the following when updating an area:
 
-According to multiple factors (including the `geostore` that is associated with the area, if the area subscribes to `fireAlerts`, `deforestationAlerts`, etc.), there might be a period of time in which the data for the area is being generated. While that is the case, the area will have `status` set to `'pending'`. Once the area data is ready, the `status` of the area will be updated to `'saved'`.
+* `geostore` with the ID of a geostore object if you are updating an area that references a geostore;
+* `wdpaid` with the ID of a protected area if you are updating an area that references a protected area;
+* `iso` object with a valid country/region/subregion if you are updating an area that references an admin country/region/subregion;
+* `use` object with valid id and name of a land use concessioned area, if you are updating an area that references a land use area.
 
-After updating an area, if it has status `saved` and if the `email` field of the area has a valid email, an email is sent to the user, to let him know the area of interest is ready to be viewed.
+**This endpoint requires authentication and, in order to PATCH an area, you need to be either the owner of the area or be an ADMIN user.**
 
-### Email substitution parameters
+### Errors for updating an area
 
-The following parameters are provided to the email service and can be used in the construction of the email:
-
-* `id` : the ID of the area.
-* `name` : the name of the area.
-* `location` : an alias for the name of the area (contains the same as the `name` parameter).
-* `subscriptions_url` : the URL to manage the areas in the frontend (example: [https://staging.globalforestwatch.org/my-gfw](https://staging.globalforestwatch.org/my-gfw)).
-* `dashboard_link` : the link to the area dashboard (example: [https://staging.globalforestwatch.org/dashboards/aoi/5d517b3fb8cfd4001061d0b2](https://staging.globalforestwatch.org/dashboards/aoi/5d517b3fb8cfd4001061d0b2)).
-* `map_link` : the "view on map" for the area (example: [https://staging.globalforestwatch.org/map/aoi/5d517b3fb8cfd4001061d0b2](https://staging.globalforestwatch.org/map/aoi/5d517b3fb8cfd4001061d0b2)).
-* `image_url` : the URL for the image associated with the area.
-* `tags` : a string containing the AOI tags, comma-separated.
-
-(`5d517b3fb8cfd4001061d0b2` is an example of an area ID).
+Error code     | Error message (example)       | Description
+-------------- | ----------------------------- | --------------
```

Comment:
Looks like this table needs updating. Messages are not parameterized and description still talks about creation.

Reply:
Done, thanks for pointing out :+1: